### DISCUSSION TITLE
Deploying simple Node.js app to Vercel

### DISCUSSION THREAD

**Discussion by AyoParadis** [vercel/community#777] [Help]
I'm trying to deploy my very simple 1-page node.js app to vercel but with no luck. I've added a vercel.json. Can anyone help?

the code: https://github.com/AyoCodess/API-for-personal-projects/blob/

Errors

![image](https://user-images.githubusercontent.com/72066389/184533540-7697b89c-a6ff-4349-91c4-26c5bc7eb52a.png)

**Comment by leerob** [comment]
Your code is private - could you share here? Does it run on your local machine with `vercel dev`?
